livre/core/CMakeLists.txt

```diff
@@ -27,10 +27,6 @@ set(LIVRECORE_HEADERS
   configuration/Configuration.h
   configuration/Parameters.h
   data/DataSource.h
-  events/EventHandler.h
```

Comment:
This PR is a suggestion:  livre/eq/EventHandler.cpp implements the Equalizer-specific event handling.